quinn-proto/src/range_set.rs

```diff
@@ -8,6 +8,292 @@ use std::{
     },
 };
 
+use tinyvec::TinyVec;
```

Comment:
Hmm, maybe this deserves a separate file? Could have `range_set/btree.rs`, `range_set/array.rs`, `range_set/tests.rs` so there's a semi-sensible place to put the tests.

Reply:
I agree and have reorganized things